packages/chakra-ui-core/src/CAvatarGroup/tests/CAvatarGroup.test.js

```diff
@@ -1,5 +1,29 @@
 import { CAvatar, CAvatarGroup } from '../..'
-import { render } from '@/tests/test-utils'
+import { render, screen, waitMs } from '@/tests/test-utils'
+
+const LOAD_FAILURE_SRC = 'LOAD_FAILURE_SRC'
+const LOAD_SUCCESS_SRC = 'LOAD_SUCCESS_SRC'
+
+beforeAll(() => {
+  process.browser = true // Mock process.browser for CAvatar created()
+})
+
+beforeAll(() => {
+  // Mock Img
+  // eslint-disable-next-line accessor-pairs
+  Object.defineProperty(global.Image.prototype, 'src', {
+    set (src) {
+      // Mock the Image.prototype.onload/onerror implementations
+      this.onload = jest.fn().mockImplementation(fn => fn)
+      this.onerror = jest.fn().mockImplementation(err => err)
```

Comment:
Same issue with mocking `onload` and `onerror` implementations.

Reply:
same as my previus comment.